packages/grafana-flamegraph/src/FlameGraphContainer.tsx

```diff
@@ -175,7 +175,7 @@ const FlameGraphContainer = ({
             <FlameGraphTopTableContainer
               data={dataContainer}
               onSymbolClick={onSymbolClick}
-              height={selectedView === SelectedView.TopTable ? 600 : undefined}
+              height={selectedView === SelectedView.TopTable || vertical ? 600 : undefined}
```

Comment:
i don't see this `height` prop having any effect, before or after. it probably gets ignored due to flex layout.